src/actions/yearn/YearnWithdrawAction.js

```diff
@@ -0,0 +1,31 @@
+const Action = require("../../Action");
+const {getAssetInfoByAddress} = require("@defisaver/tokens");
+const { getAddr } = require('../../addresses.js');
+
+/**
+ * YearnWithdrawAction - Supply token to a Yearn vault
```

Comment:
`YearnWithdrawAction - Supply token to a Yearn vault` - I'm getting some mixed signals here